core/camera.js

```diff
@@ -86,15 +86,15 @@ class Camera {
 
     objs.forEach((o) => {
       if (this.inView(o)) {
-        o.render(Rectangle.offset(o, this.offsetX, this.offsetY));
+        o.render.call(o, Rectangle.offset(o, this.offsetX, this.offsetY));
```

Comment:
Isn't `o.render` the same then `o.render.call(o, ...)`. I mean, you're passing `o` as context that has the same effect than calling `render` from the o's instance. Right?